.github/workflows/ps-ce-sdk-psgallery-publish.yaml

```diff
@@ -0,0 +1,26 @@
+# **************************************************************************
+#  Copyright (c) Cloud Native Foundation.
+#  SPDX-License-Identifier: Apache-2.0
+# **************************************************************************
+
+name: Publish to PSGallery
+
+on:
+  create:
```

Comment:
We typically use `push` in other repos. Not sure how `create` behaves when someone pushes a tag via `git` CLI vs UI. Did you verify pushing commits would also trigger this workflow?

Reply:
Create works on both pushing a tag and creating the tag from GUI